packages/fxa-settings/src/pages/ResetPassword/CompleteResetPassword/container.tsx

```diff
@@ -102,10 +89,16 @@ const CompleteResetPasswordContainer = ({
         replace: true,
         state: {
           email,
-          fancyBannerSuccessMessage: {
-            heading: successBannerMessageHeading,
-            message: successBannerMessageMessage,
-          },
+          successBanner: {
+            localizedSuccessBannerHeading: ftlMsgResolver.getMsg(
+              'reset-password-complete-banner-heading',
+              'Your password has been reset.'
+            ),
+            localizedSuccessBannerDescription: ftlMsgResolver.getMsg(
+              'reset-password-complete-banner-message',
+              'Don’t forget to generate a new account recovery key from your Mozilla account settings to prevent future sign-in issues.'
```

Comment:
Do you know when we use "sign-in" vs "sign in"? I just assumed we should never have the dash, but now I'm not sure?

Reply:
`sign in` is the verb, and `signin` or `sign-in` (both are  accepted) are nouns or adjectives.